calculator.html

```diff
@@ -72,14 +71,13 @@
           </li>
         </ul>
       </form>
-      <a href="#" data-bind="click: removeCalculator, visible: calculators().length > 1">Remove</a>
+      <a href="#" data-bind="click: $parent.removeCalculator, visible: $parent.calculators().length > 1">Remove</a>
     </div>
   </div>
   <div class="add-calculator-div">
-    <button class="add-calculator" data-bind="click: addCalculator">  Add Calculator </button>
+    <button class="add-calculator" data-bind="click: addCalculator">Add Calculator</button>
   </div>
-  <script type="text/javascript" src="lib/calculator.js"></script>
-  <script type="text/javascript" src="app.js"></script>
+  <script src="node_modules/requirejs/require.js" data-main="require-config.js"></script>

```

Comment:
We no longer reference code files. We reference require.js. Once it has loaded, it will load the file specified by the data-main attribute. See below for more info on what that does.